cmd/create.go

```diff
@@ -16,8 +16,8 @@ func init() {
 	root.AddCommand(createCmd)
 	createCmd.Flags().BoolP("confirm", "c", false, "Prompt to confirm all configuration options (Env: $FUNC_CONFIRM)")
 	createCmd.Flags().StringP("runtime", "l", bosonFunc.DefaultRuntime, "Function runtime language/framework. Available runtimes: "+utils.RuntimeList()+" (Env: $FUNC_RUNTIME)")
-	createCmd.Flags().StringP("templates", "", filepath.Join(configPath(), "templates"), "Path to additional templates (Env: $FUNC_TEMPLATES)")
-	createCmd.Flags().StringP("trigger", "t", bosonFunc.DefaultTrigger, "Function trigger. Available triggers: 'http' and 'events' (Env: $FUNC_TRIGGER)")
+	createCmd.Flags().StringP("packages", "p", filepath.Join(configPath(), "packages"), "Path to additional template packages (Env: $FUNC_PACKAGES)")
```

Comment:
Minor concern, `-p` is being used for path with the other commands, this might get a little bit confusing from UX. But not sure if it is really an issue or just my nit picking 🤷‍♂️

Reply:
We could bike-shed a bit and rename the whole parameter 'repositories'... Or I can just change it.

I think for now I'll just change it to 'a' for "Additional Packages".  That seems reasonable?

Reply:
@lkingland I think it's maybe worth bike shedding a bit. See https://github.com/boson-project/func/pull/375#discussion_r648669324